.github/workflows/bms_prod_ci_cd.yml

```
@@ -0,0 +1,145 @@
+on:
+  push:
+    branches:
+      - bms-prod
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)